netlify.toml

```diff
@@ -1,5 +1,5 @@
 [context.production.environment]
-  REACT_APP_SANITY_TAG="default"
+  REACT_APP_SANITY_TAG = "default"
```

Comment:
does the whitespace here matter?

Reply:
I don't think so, but this makes it consistent with the rest of the file